### PR TITLE
WIP: interconnect operator installation

### PR DESCRIPTION
**Pull request by adtandale**

### CODE REVIEW

**Comment by ascerra** [comment]
@adtandale It looks like you are not a member of the openshift org please follow this prerequisite guide to become a member. https://cspi-qe.gitbook.io/ocp-ci-onboarding/onboarding/onboarding_guide/prerequisites_guide#github-user-readiness

**Comment by adtandale** [comment]
> You are right @ascerra,I had created ticket - https://issues.redhat.com/browse/DPP-12310 seems in progress, thanks.

File: ci-operator/config/skupper/skupper-operator/skupper-skupper-operator-4.11-candidate.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+binary_build_commands: make manager
```
[the remark this answers is not in the document]
Comment (adtandale) [reply ci-operator/config/skupper/skupper-operator/skupper-skupper-operator-4.11-candidate.yaml:1]: okay, removing the file

File: ci-operator/jobs/interconnect-qe/interconnect-tests/interconnect-qe-interconnect-tests-main-periodics.yaml [hunk 2]
```diff
@@ -6,17 +6,16 @@ periodics:
   decoration_config:
     skip_cloning: true
   extra_refs:
-  - base_ref: main
```
Comment (ascerra) [inline ci-operator/jobs/interconnect-qe/interconnect-tests/interconnect-qe-interconnect-tests-main-periodics.yaml]: We should never need to manually update a prow job config.

These get automatically generated and updated based on changes we make in our configs. So if the job doesn't have the correct values then we need to run `make update` to fix the values.

I did notice pj-rehearse didn't recognize something and I'm guessing this is why we are attempting to edit the jobs itself?

**Comment by adtandale** [comment]
/pj-rehearse periodic-ci-rh-messaging-qdr-shipshape-main-interconnect-ocp4.13-lp-interop-interconnect-amp-interop-aws

**Comment by adtandale** [comment]
/pj-rehearse periodic-ci-rh-messaging-qdr-shipshape-main-interconnect-ocp4.13-lp-interop-interconnect-amp-interop-aws

File: ci-operator/jobs/interconnect-qe/interconnect-tests/interconnect-qe-interconnect-tests-main-periodics.yaml (hunk 2 above)
Comment (adtandale) [reply ci-operator/jobs/interconnect-qe/interconnect-tests/interconnect-qe-interconnect-tests-main-periodics.yaml]: New changes are made in files.

**Comment by adtandale** [comment]
/pj-rehearse periodic-ci-rh-messaging-qdr-shipshape-main-interconnect-ocp4.13-lp-interop-interconnect-amp-interop-aws

File: ci-operator/config/rh-messaging/qdr-shipshape/rh-messaging-qdr-shipshape-main__interconnect-ocp4.13-lp-interop.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      OPERATORS: |
+        [
+            {"name": "amq-interconnect-operator", "source": "redhat-operators", "channel": "!default", "install_namespace": "default", "target_namespaces": "default"}
+        ]
+      SPOT_INSTANCES: "true"
```
Comment (calebevans) [inline ci-operator/config/rh-messaging/qdr-shipshape/rh-messaging-qdr-shipshape-main__interconnect-ocp4.13-lp-interop.yaml:35]:
```suggestion
      SPOT_INSTANCES: "false"
```

Your last error seemed to be an AWS error with spot instances. Lets try setting this to false and re-run to see if we can get it to pass for you

Comment (adtandale) [reply ci-operator/config/rh-messaging/qdr-shipshape/rh-messaging-qdr-shipshape-main__interconnect-ocp4.13-lp-interop.yaml]: okay, pushed the change, lets see. Thanks!

**Comment by adtandale** [comment]
/pj-rehearse periodic-ci-rh-messaging-qdr-shipshape-main-interconnect-ocp4.13-lp-interop-interconnect-amp-interop-aws

**Comment by adtandale** [comment]
/pj-rehearse periodic-ci-rh-messaging-qdr-shipshape-main-interconnect-ocp4.13-lp-interop-interconnect-amp-interop-aws

**Comment by adtandale** [comment]
/pj-rehearse periodic-ci-rh-messaging-qdr-shipshape-main-interconnect-ocp4.13-lp-interop-interconnect-amp-interop-aws

**Comment by adtandale** [comment]
/pj-rehearse periodic-ci-rh-messaging-qdr-shipshape-main-interconnect-ocp4.13-lp-interop-interconnect-amp-interop-aws

**Pull request closed without merging**